CONTRIBUTING.org

```diff
@@ -0,0 +1,213 @@
+* Contributing a new recipe to MELPA
+
+MELPA consists of /recipes/, each describing one /package/ in a
+dedicated repository. This document describes how to propose a new
+package for inclusion into MELPA.
+
+Three contributor roles are involved in the maintenance of MELPA :
+
+- MELPA maintainers :: They are responsible for reviewing new recipes
+     and associated packages as well as keeping the system working.
+
+- Recipe authors :: They responsible for submitting good recipes and
+                    maintaining them when the package is changed
+                    (e.g., renamed or deprecated).
+
+- Package author :: They are responsible for writing Emacs Lisp
+                    packages of good quality.
+
+We advise package author to also take the recipe author role when
+possible.
+
+In the following, package and recipe authors will find guidelines to
+help them prepare a package and its recipe for inclusion into
+MELPA. Please follow the guidelines as closely as possible to speed up
+the inclusion process. Guidelines are not strict rules though and they
+can sometimes be bypassed: in these cases, the package author is
+required to explain why a guideline was not followed.
+
+** What to know before opening a pull request
+
+New recipe submissions should adhere to the following guidelines:
+
+- One recipe per pull request :: If you have several packages to
+     submit, open several pull requests. If the packages are very
+     closely related and maintained in the same repository (see below
+     for why it might not be a good idea though), you might open only
+     one pull request for them all; in this case, keep adding one
+     recipe per commit though.
+
+- Reasonably innovative package :: MELPA provides a curated set of
+     Emacs Lisp packages, not an exhaustive list of every single Emacs
+     Lisp file ever created. As such, MELPA maintainers tend to only
+     accept packages not duplicating existing packages. Please try to
```

Comment:
By default, MELPA maintainers will reject packages that duplicate functionality provided by existing packages.